src/routes/committee/schedule/availability/[rehearsalId]/+page.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+	import type { availability } from '$lib/types/domain/availability'
+	import type { rehearsal } from '$lib/types/domain/rehearsal'
+	import type { user } from '$lib/types/domain/user'
+	import {
+		Button,
+		Grid,
+		Modal,
+		StructuredList,
+		StructuredListCell,
+		StructuredListHead,
+		StructuredListRow
+	} from 'carbon-components-svelte'
+	import { Chat } from 'carbon-icons-svelte'
+
+	export let data: {
+		users: { [id: string]: user }
+		availabilities: { [id: string]: availability }
+		rehearsal: rehearsal
+	}
+	let reasonText: string = ''
+	let openReason: boolean = false
+</script>
+
+<h1>Rehearsals {data.rehearsal.startTime.toDate().toDateString()}</h1>
+<Grid>
+	<StructuredList>
+		<StructuredListHead>
+			<StructuredListRow head>
+				<StructuredListCell head>Participant</StructuredListCell>
+				<StructuredListCell head>Availability</StructuredListCell>
+			</StructuredListRow>
+		</StructuredListHead>
+		{#each Object.entries(data.users) as [id, user]}
+			<StructuredListRow>
+				<StructuredListCell>
+					{user.name}
+				</StructuredListCell>
+				<StructuredListCell>
+					{#if id in data.availabilities}
+						{#if data.availabilities[id].available}
+							{data.availabilities[id].startTime.toDate().getHours()}:{String(
+								data.availabilities[id].startTime.toDate().getMinutes()
+							).padStart(2, '0')} -
+							{data.availabilities[id].endTime.toDate().getHours()}:{String(
+								data.availabilities[id].endTime.toDate().getMinutes()
+							).padStart(2, '0')}
```

Comment:
Hier beter een utility function voor declaren die bijv data.availabilities[id].endTime.toDate() pakt en de format teruggeeft. 

Of mooier nog, je gebruikt toLocaleDateString als dat kan:
https://stackoverflow.com/questions/3552461/how-do-i-format-a-date-in-javascript
but not sure